Content.Server/Clothing/Components/ClothingComponent.cs

```diff
@@ -101,6 +107,11 @@ private bool TryEquip(InventoryComponent inv, Slots slot, IEntity user)
                 return false;
             }
 
+            if (EquipSound != null)
```

Comment:
equipsound isnt marked as nullable (should be `SoundSpecifier?`) so either do that or remove this check and add `required: true`